Dockerfile

```diff
@@ -7,6 +7,7 @@ COPY --from=thrift /usr/local/bin/thrift /usr/local/bin/thrift
 WORKDIR /src
 
 COPY requirements*.txt ./
+RUN apt-get -y update && apt-get -y install librdkafka-dev && rm -rf /var/lib/apt/lists/*
```

Comment:
Can we move this new apt line above the COPY so that we don't need to reinstall apt packages every time a requirement changes? Maybe it can go directly under the `FROM python:3.12` line?

Reply:
Good call, I'll move this up.